src/Block/AdminSearchBlockService.php

```diff
@@ -42,13 +42,21 @@ class AdminSearchBlockService extends AbstractBlockService
     protected $searchHandler;
 
     /**
-     * NEXT_MAJOR: Change signature for (Environment $twig, Pool $pool, SearchHandler $searchHandler).
+     * NEXT_MAJOR: Change to string.
+     *
+     * @var string|null
+     */
+    private $emptyBoxesOption;
+
+    /**
+     * NEXT_MAJOR: Change signature for (Environment $twig, Pool $pool, SearchHandler $searchHandler, string $emptyBoxesOption).
      *
      * @param Environment|string        $twigOrName
      * @param Pool|EngineInterface|null $poolOrTemplating
      * @param SearchHandler|Pool        $searchHandlerOrPool
+     * @param string|SearchHandler      $emptyBoxesOptionOrSearchHandler
```

Comment:
Can the string take any value, or is it a finite list? If that's the later we should maybe have `@phpstan-param 'show'|'other_value'|SearchHandler`

Reply:
I have absolutely no idea 😅

I found this: https://github.com/sonata-project/SonataAdminBundle/pull/4334
So I updated the phpdoc.